lib/diagnostic.js

```diff
@@ -14,12 +14,22 @@
 'use strict';
 
 const fs = require('fs'); // jshint ignore: line
+const inFile = process.argv[2];
 
-const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
+const sumLines = ('integers.txt', callback) => {
+  fs.readFile('/data/integers.txt', 'utf8', (error, data));
+  if (error){
+    console.error(error.stack);
+    return;
+  }
+  let dataSum = data.reduce(function(a, b){
```

Comment:
You have the right idea! But! At this point, data is just a long string of characters and carriage returns. What does reduce take? It takes an array.